### PR TITLE
added support for loading QUALapprox into VET

### PR DESCRIPTION
**Pull request by kcibul**
Part of resolving https://github.com/broadinstitute/dsp-spec-ops/issues/244

Loading site-level QualApprox into VET... usage will be in a different PR

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/VetFieldEnum.java [hunk 1]
```diff
@@ -167,6 +167,16 @@ public String getColumnValue(final VariantContext variant) {
         }
     },
 
+    QUALapprox { // Required
+        public String getColumnValue(final VariantContext variant) {
+            String out = getAttribute(variant, "QUALapprox", null);
+            if (out == null) {
+                throw new UserException("Cannot be missing required value for QUALapprox at site: " + variant.toString());
```
Comment (mmorgantaylor) [inline src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/VetFieldEnum.java:174]: neat. out of curiosity - is there documentation somewhere of the input requirements? is QUALapprox always present in gvcfs? also thinking more broadly about the method of reblocking required, etc.

**Review by mmorgantaylor — APPROVED** [review]
looks good to me! general question about documentation but unless there's existing documentation to update, no changes needed from me

**Review by ahaessly — APPROVED** [review]
👍

File: src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/VetFieldEnum.java (hunk 1 above)
Comment (kcibul) [reply src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/VetFieldEnum.java:174]: It's more that we know the downstream tools require this annotation, but it's produced by GATK Reblocking (which is the real prerequisite).

**Pull request merged**